ui/app/src/views/ViewDashboardList.tsx

```diff
@@ -66,18 +109,57 @@ function RenderDashboardList() {
     );
   });
 
-  return <Box>{accordions}</Box>;
+  return (
+    <>
+      <Box>{accordions}</Box>
+      <DeleteProjectDialog
+        name={projectToDelete}
+        open={openDeleteProjectDialog}
+        onClose={closeDeleteProjectConfirmDialog}
+        onConfirm={triggerProjectDeletion}
+      />
+    </>
+  );
 }
 
 function ViewDashboardList() {
+  const [openCreateProjectDialogState, setOpenCreateProjectDialogState] = useState(false);
+
+  const { successSnackbar, exceptionSnackbar } = useSnackbar();
+  const navigate = useNavigate();
+  const createProjectMutation = useCreateProjectMutation();
+  const triggerProjectCreation = function (name: string) {
+    createProjectMutation.mutate(name, {
+      onSuccess: (entity: ProjectModel) => {
+        successSnackbar(`project ${entity.metadata.name} was successfully created`);
+        navigate(`/projects/${entity.metadata.name}`);
+      },
+      onError: (err) => {
+        exceptionSnackbar(err);
+      },
+    });
+  };
+
   return (
     <Container maxWidth="md" sx={{ marginY: 2 }}>
-      <Typography variant="h2" mb={2}>
-        Dashboards
-      </Typography>
+      <Stack direction="row" alignItems="center" justifyContent="space-between">
+        <Stack direction="row" alignItems="center" justifyContent="start" gap={1} my={2}>
+          <Typography variant="h2">All Dashboards</Typography>
+        </Stack>
+        <Stack direction="row" alignItems="center" justifyContent="end" gap={1} my={2}>
+          <Button variant="outlined" size="small" onClick={() => setOpenCreateProjectDialogState(true)}>
+            ADD PROJECT
```

Comment:
```suggestion
            Add Project
```

Reply:
I just applied the design from the designer hehe 
![image](https://user-images.githubusercontent.com/10258608/214533003-4542a9d3-7f58-4003-9b7f-9b76af87a53b.png)

But I can do this yeah if you think it is more consistent with the rest of the views.